socorro/schemas/processed_crash.schema.yaml

```diff
@@ -2181,57 +2181,82 @@ properties:
     type: ["string", "null"]
     permissions: ["public"]
   android_board:
-    description: The board used by the Android device.
+    description: >
+      The name of the underlying board used by the Android device.
```

Comment:
The descriptions are copied over from [CrashAnnotations.yaml](https://searchfox.org/mozilla-central/source/toolkit/crashreporter/CrashAnnotations.yaml).